packages/sra-report/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "lib"
+    },
+    "include": [
+        "./src/**/*",
+        "../../node_modules/web3-typescript-typings/index.d.ts",
+        "../../node_modules/ethers-typescript-typings/index.d.ts"
```

Comment:
Remove both of these. Neither is being used by this package.

Reply:
We need to look into this, any TS project that has 0x.js as a dependency will not compile w/o them

Reply:
@BMillman19 can you double check that this is still the case? Fixed some related issue over the weekend.